cmd/broker/ingress/kodata/HEAD

```diff
@@ -0,0 +1 @@
+ref: refs/heads/brokeringress
```

Comment:
The files in kodata should be symlinks, not actual files. Take a look at the other directories to see where they should link.

Reply:
Oh didn't know that. Just did a `cp` without preserving links...

Please take a look now.

Reply:
Looks good now, thanks!